### PR TITLE
SB - remove support for Gardener `adminkubeconfig`

### PR DESCRIPTION
**Pull request by jaroslaw-pieszka**
**Description**

Changes proposed in this pull request:

- support for gardener admin kubeconfig removed

**Related issue(s)**

### CODE REVIEW

File: docs/user/05-60-kyma-bindings.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ X-Broker-API-Version: 2.14
 }
 ```
 
-The Broker returns a kubeconfig file in the response body. The kubeconfig file contains the necessary information to access the managed Kyma cluster. By default, KEB uses [`shoots/adminkubeconfig`](https://github.com/gardener/gardener/blob/master/docs/usage/shoot_access.md#shootsadminkubeconfig-subresource) subresources to generate a kubeconfig that uses certificates to authenticate its user. To customize the format of the returned kubeconfig, use the `parameters` field of the request body:
+The Broker returns a kubeconfig with a JWT token used as a user authentication mechanism. The token is generated using Kubernetes TokenRequest attached to a ServiceAccount, ClusterRole, and ClusterRoleBinding, all named `kyma-binding-{{binding_id}}`. Such an approach allows for modifying the permissions granted to the kubeconfig.
+You can specify the duration for which the generated kubeconfig is valid by providing the `expiration_seconds` in the `parameter` field of the request body.
```
Comment (IwonaLanger) [inline docs/user/05-60-kyma-bindings.md:19]:
```suggestion
To specify the duration for which the generated kubeconfig is valid, provide the **expiration_seconds** in the `parameter` object of the request body. 
```

**Review by IwonaLanger — APPROVED** [review]

**Comment by MarekMichali** [comment]
`GardenerBindingManager` struct and related code should be also removed,

**Review by IwonaLanger — APPROVED** [review]

**Comment by MarekMichali** [comment]
`BindingDTO` needs adjustment

**Comment by MarekMichali** [comment]
`NewBind` still has `gardenerClient` as an argument.

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**